### ISSUE TITLE
Fix multi-line double-slash command comments

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#5348]
Refs: https://github.com/stylelint/stylelint/issues/4942#issuecomment-841786453 & https://github.com/stylelint/stylelint/pull/5346#discussion_r645994299

This issue only impacts users who are using Less and multi-line double-slash command comments, e.g.:

```scss
// stylelint-disable color-hex-case
// --
// description

a {color: #FFF; }
```

I doubt many, if any, users do this as Less isn't popular and support for multi-line double-slash command comments is a relatively new feature. As the workaround is to simply switch to `/*` style comments for these multi-line commands, I'd say this is a low priority. I'll add it to the `14.0.0` umbrella issue in case it does impact someone and they want to pick it up, but I don't see it as a blocker to releasing.

**Comment by nwalters512** [comment]
Wanted to add a +1 to this. We use a lot of such comments, and we started seeing Stylelint report issues in really weird places. For instance, a block of `//` comments in the middle of a file caused a lint error at the end of the file to be reported on a token in the middle of the comment. Here's a stripped-down version of that file illustrating the issue:

<img width="714" alt="Screen Shot 2021-08-09 at 3 46 41 PM" src="https://user-images.githubusercontent.com/1476544/128783712-5495e5d1-6a08-4ee2-a4e1-4a296f03a611.png">

I finally managed to figure out that the common link between all files with weird errors was the use of multi-line `//` comments and trace it back to this GitHub issues, but it was a reallllly confusing 30 minutes spent banging my head against the table.
